aiida/cmdline/params/options/__init__.py

```diff
@@ -451,7 +452,43 @@ def decorator(command):
 
 FREQUENCY = OverridableOption('-F', '--frequency', 'frequency', type=click.INT)
 
-VERBOSE = OverridableOption('-v', '--verbose', is_flag=True, default=False, help='Be more verbose in printing output.')
+VERBOSE = OverridableOption('-v', '--verbose', count=True, help='Print more verbose output.')
```

Comment:
Note: This option will be removed/replaced (on the to-do list).

Reply:
Do you mean you plan to remove this in this PR?

Reply:
Yes, I will.